### PR TITLE
link to wiki's Sway entry, nontrivial info there

### PR DESCRIPTION
**Pull request by aaronchall**
###### Description of changes
Link to wiki's Sway entry as this information in the manual is otherwise not very helpful at accomplishing what it implies is a trivial modification. This link would have saved me quite a lot of time.

###### Things done

Literally nothing as it's one line of doc.

### CODE REVIEW

**Review by teutat3s — APPROVED** [review]

**Review by winterqt — CHANGES_REQUESTED** [review]
Please run `nixos/doc/md-to-db.sh`.

**Comment by aaronchall** [comment]
> Please run `nixos/doc/md-to-db.sh`.
Is that to me? If not who to? If to me... I'll try! :)

**Comment by winterqt** [comment]
@aaronchall Yup, that was to you. Run it, squash, and force push. That's why the manual build is failing -- you have to update both the source Markdown files and the generated DocBook files.

**Review by teutat3s — APPROVED** [review]
```
❯ nix-build nixos/release.nix -A manual.x86_64-linux
```
builds fine here.

File: nixos/doc/manual/configuration/wayland.chapter.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ xdg.portal.wlr.enable = true;
 
 and configure Pipewire using
 [](#opt-services.pipewire.enable)
-and related options.
+and related options (see the [wiki's Sway entry](https://nixos.wiki/wiki/Sway)).
```
Comment (mweinelt) [inline nixos/doc/manual/configuration/wayland.chapter.md:27]: Not that wiki.

**Review by mweinelt — CHANGES_REQUESTED** [review]

**Pull request closed without merging**